CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+## Contributing
+
+1. Familiarize yourself with the codebase by reading the [docs](docs).
+1. Create a new issue before starting your project so that we can keep track of
+   what you are trying to add/fix. That way, we can also offer suggestions or
+   let you know if there is already an effort in progress.
+1. Fork this repository.
+1. The [README](README.md) has details on how to set up your environment.
+1. Create a _topic_ branch in your fork based on the correct branch (usually the **develop** branch, see [Branches section](#branches) below). Note, this step is recommended but technically not required if contributing using a fork.
+1. Edit the code in your fork.
+1. Sign CLA (see [CLA](#cla) below)
+1. Send us a pull request when you are done. We'll review your code, suggest any
+   needed changes, and merge it in.
```

Comment:
Should this contributing section be modified for this project?  Something like, "when adding or modifying an automated test, this is how you do it..."

Reply:
Done. (added an extra step about running against the develop branch of the extension's repo)